src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SnsService.kt

```diff
@@ -73,10 +73,7 @@ class SnsService(hmppsQueueService: HmppsQueueService, private val objectMapper:
           ).also { log.info("Published event $payload to outbound topic") }
       )
     } catch (e: Throwable) {
-      // throw PublishEventException("Failed to publish Event $payload.eventType to $TOPIC_ID", e)
-
-      // Note: Silently fail until VB-671 is implemented
-      log.debug("Failed to publish Event $payload.eventType to $TOPIC_ID", e)
+      throw PublishEventException("Failed to publish Event $payload.eventType to $TOPIC_ID", e)
```

Comment:
good call

Reply:
Need to ask our new test guy if there's any way he can test this stuff